benchmarks/add_benchmark.c

```diff
@@ -117,7 +117,7 @@ void run_test(uint32_t spanlen, uint32_t intvlen, double density,
     for (int p = 0; p < num_passes; p++) {
         roaring_bitmap_t *r = roaring_bitmap_create();
         RDTSC_START(cycles_start);
-        roaring_bulk_context_t context = {0};
```

Comment:
These are actually all okay: `{0}` acts as a "universal zero initializer":

> If there are fewer initializers in a list than there are members of an aggregate, the remainder of the aggregate shall be initialized implicitly the same as objects that have static storage duration.
> [...]
> If an object that has static storage duration is not initialized explicitly, it is initialized implicitly as if every member that has arithmetic type were assigned 0 and every member that has pointer type were assigned a null pointer constant.

Using just `{0}` is shorter, doesn't have to change if the structure changes, and once you know the idiom, clearer imo.

Reply:
@Dr-Emann C++ compilers issue warnings for each instance.

Our C code is meant to be compilable both as C and C++. 

The way to do it in C++ would be just {}, but it does not seem to work in C11.

Reply:
That's.. really annoying, it's a pretty standard idiom. 😞 

Not super happy about it, but makes sense 👍

Reply:
See https://godbolt.org/z/fsKsP6qf3

Reply:
Other way would be to use a `ZERO_INIT` macro that's either `{}` or `{0}`, but I think I'd rather just add the extra zeros as needed.

Bleh.

Reply:
C23 https://stackoverflow.com/questions/17589533/is-an-empty-initializer-list-valid-c-code

Reply:
Basically, I don't want the code to throw a ton of warnings when building in C++ as some users just panic and report it as a bug. But I am open to any pragmatic solution.

Reply:
Yeah, I doing what's in this pr is the least bad option from my perspective.